modules/proc-configuring-container-registry-authn.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following:
+//
+// @smccarthy-ie - Getting Started with Camel Kafka Connector
+//
+// Attributes used:
+//
+// {ProductName}
+// {OCPVersion}
+
+
+[id="configuring-container-registry-authn_{context}"]
+= Configuring authentication with registry.redhat.io
+
+You must configure authentication with `registry.redhat.io` before you can deploy {ProductName} container images on OpenShift.
```

Comment:
I'd change this from 'you must' to a more abstracty type sentence.  'Configure authentication with ....'